### PR TITLE
Show more of the url (#6348)

### PR DESCRIPTION
**Pull request by qifahs**
Dialog service starts truncating words longer than 80 characters and adds ellipses to it. To show more of the URL, we can show about 80 characters in the beginning and 80 in the end, with ellipses in between. Changed truncation logic to handle this. 

Here is the before/after experience for opening this URL: azuredatastudio://microsoft.notebook/open?url= https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2Fazuredatastudio%2Fmaster%2Fsamples%2FnotebookSamples%2FSampleTSQLNotebook.ipynb

Before:
image

After:
image

### CODE REVIEW

File: src/vs/workbench/services/extensions/common/inactiveExtensionUrlHandler.ts [hunk 1]
```diff
@@ -109,8 +109,14 @@ class ExtensionUrlHandler implements IExtensionUrlHandler, IURLHandler {
 		if (!confirmed) {
 			let uriString = uri.toString();
 
-			if (uriString.length > 40) {
-				uriString = `${uriString.substring(0, 30)}...${uriString.substring(uriString.length - 5)}`;
+			// Dialog service starts truncating words longer than 80 characters and adds ellipses to it.
+			// To show more of the URL, we can show about 80 characters in the beginning and 80 in the end, with ellipses in between.
+			if (uriString.length > 80) {
```
Comment (kburtram) [inline src/vs/workbench/services/extensions/common/inactiveExtensionUrlHandler.ts]: whenever we edit code under the `vs` folder we add a comment like `// {{SQL CARBON EDIT}` around the change.  This helps a lot when merging in new VS Code changes so we can easily see we've changed.  Could you please add that for this change?

Comment (qifahs) [reply src/vs/workbench/services/extensions/common/inactiveExtensionUrlHandler.ts]: Sure thing. Just added it.

**Comment by Charles-Gagnon** [comment]
FYI when you create a PR it's helpful to tag the original issue in the description so they're automatically linked by GitHub. And if you add a [special keyword](https://help.github.com/en/articles/closing-issues-using-keywords) then it'll even auto-close the issue once the PR is merged.

**Comment by Charles-Gagnon** [comment]
> 				type: 'question'

Might be useful to have another button "Copy Link" so users can copy the entire link if they want to. Don't think dialog service supports this though unfortunately so you'd have to use a normal dialog if so. This is fine as a general improvement though if that's going to be a bunch of extra work.

---
Refers to: src/vs/workbench/services/extensions/common/inactiveExtensionUrlHandler.ts:131 in 1ed6f63. [](commit_id = 1ed6f63bc08d5b10b9df94791d2ae6f8cf5e8e88, deletion_comment = False)

**Pull request closed without merging**